src/storage/index.js

```diff
@@ -13,22 +13,30 @@ const s3 = new aws.S3({
   credentials: config.s3.credentials,
 });
 
-const getFileUpload = (fileTag, maxFileSize) => {
-  const fileUpload = multer({
-    storage: multerS3({
-      s3,
-      bucket: config.s3.bucket,
-      acl: 'public-read',
-      key: (req, file, next) => {
-        next(null, `portal/profiles/${req.user.uuid}${path.extname(file.originalname)}`);
-      },
-    }),
+const uploadToS3 = async (folder, file, uuid) => {
+  const params = {
+    ACL: 'public-read',
+    Body: file.buffer,
+    Bucket: config.s3.bucket,
+    Key: `portal/${folder}/${uuid}${path.extname(file.originalname)}`,
+  };
+  const coverUrl = await s3.upload(params).promise()
+    .then((data) => data.Location)
+    .catch((error) => {
+      log.warn(`Failed to upload profile picture to S3: ${error}`);
+    });
+
+  return coverUrl;
+};
+
+const getFile = (maxFileSize) => {
```

Comment:
1. `getFile` is a misnomer since `multer` is middleware that processes the image blob and stores it on disk or in a memory buffer; rename to something more accurate like `bufferImageBlob` or something.
2. The `multer` object doesn't specify a [`storage`](https://www.npmjs.com/package/multer#multeropts) engine. In its current form, `getFileUpload` specifies a `multer-s3` object as its `storage` engine, so you'll want to use the [memory storage engine](https://www.npmjs.com/package/multer#memorystorage) here.